.mergify.yml

```
@@ -0,0 +1,38 @@
+queue_rules:
+  - name: main
+    batch_size: 3
+    batch_max_wait_time: 4h
+    checks_timeout: 12h
+    merge_method: squash
+    update_method: rebase
+    merge_conditions:
+      - check-success = test-suite-e2e-tests/fhevm-e2e-test (bpr)
+    queue_conditions:
+      - base = main
+      - "#approved-reviews-by >= 1"
+      - label!=do-not-merge
+      - status-success = coprocessor-db-migration-docker-build/build (bpr)
```

```suggestion
      - check-success = coprocessor-db-migration-docker-build/build (bpr)
```
`status-` is deprecated for a long time